### PR TITLE
[python-frontend] Implement Operational Modelss that are imported

### PR DESCRIPTION
**Pull request by ivomachado**
Start with a very small math model implementation to showcase the feature.

This allows a python file to import `math` and import our OM for it.

### CODE REVIEW

**Review by lucasccordeiro — APPROVED** [review]

File: src/python-frontend/parser.py [hunk 1]
```diff
@@ -140,11 +148,13 @@ def main():
...
     # Iterate over all .py files in the directory
     for python_file in glob.glob(os.path.join(models_dir, "*.py")):
-        with open(python_file) as model:
-            model_tree = ast.parse(model.read())
-            filename = os.path.basename(python_file)
-            # Generate JSON from AST for the memory models.
-            generate_ast_json(model_tree, filename, None, output_dir)
+        filename = os.path.basename(python_file)
+        module_name = filename[:-3]
+        if not is_imported_model(module_name):
```
Comment (brcfarias) [inline src/python-frontend/parser.py]: Could change to “if is_imported continue”?

File: src/python-frontend/models/math.py
Comment (brcfarias) [inline src/python-frontend/models/math.py:1]: Is it possible to add a math function to this initial version?

**Review by brcfarias — APPROVED** [review]

File: src/python-frontend/models/math.py
Comment (ivomachado) [reply src/python-frontend/models/math.py:1]: yes, it is. I'm going to add floor, ok?

**Comment by lucasccordeiro** [comment]
Thanks for submitting this PR, @ivomachado.

**Pull request merged**